src/cpufreq.c

```diff
@@ -25,8 +25,71 @@
 #include "common.h"
 #include "plugin.h"
 
+#define MAX_AVAIL_FREQS 20
```

Comment:
Why exactly this value is used?

Reply:
its used to create an array of frequencies that core can spend time in. Must be fixed size as its an element of another array allocated using calloc.

Reply:
Why exactly 20 ? What this value is based on?